lib/Tool/Console.php

```diff
@@ -115,6 +139,26 @@ public static function getExecutable(string $name, bool $throwException = false)
         return false;
     }
 
+    protected static function checkComposite(string $process): bool
+    {
+        return self::checkConvert($process);
+    }
+
+    protected static function checkConvert(string $executablePath): bool
+    {
+        try {
+            $process = new Process([$executablePath, '--help']);
+            $process->run();
+            if (strpos($process->getOutput() . $process->getErrorOutput(), 'imagemagick.org') !== false) {
+                return true;
+            }
+        } catch (\Exception $e) {
+            // noting to do
+        }
+
+        return false;
+    }
+
```

Comment:
This is not used, right?
```suggestion
```

Reply:
Right, as far as I can see they are not used.